jwst/lib/tests/test_fgs_pointing.py

```diff
@@ -0,0 +1,72 @@
+"""Test suite for ensuring correct FGS pointing"""
+import logging
+from numpy import array
+
+from ...datamodels import Level1bModel
+from .. import set_telescope_pointing as stp
+
+
+# Set logging for the module to be tested.
+logger = logging.getLogger(stp.__name__)
+logger.setLevel(logging.INFO)
+handler = logging.StreamHandler()
+handler.setLevel(logging.INFO)
+logger.addHandler(handler)
+
+# Define minimal model meta structure
+WCS_META = {
+    'meta': {
+        'exposure': {
+            'type': 'FGS_ACQ1',
+        },
+        'wcsinfo': {
+            'cdelt1': 1.90364333333333e-05,
+            'cdelt2': 1.94607944444444e-05,
+            'crpix1': 1024.5,
+            'crpix2': 1024.5,
+            'crval1': 0,
+            'crval2': 0,
+            'ctype1': 'RA---TAN',
+            'ctype2': 'DEC--TAN',
+            'cunit1': 'deg',
+            'cunit2': 'deg',
+            'dec_ref': 0.0,
```

Comment:
I believe we had decided to not include the RA_REF, DEC_REF, and ROLL_REF keyword values for guide mode exposures, so you don't really need these in the list (unless something in an interface somewhere depends on them being there).

Reply:
Sure, just copied from another test file, but will remove.